app/models/order.rb

```diff
@@ -1,6 +1,7 @@
 class Order <ApplicationRecord
-  validates_presence_of :name, :address, :city, :state, :zip
+  validates_presence_of :name, :address, :city, :state, :zip, :status
```

Comment:
Do we know yet what the alternative values of the `status` attribute will be beyond `pending`?
This might me another use case for another enum.

Reply:
packaged, pending, shipped, cancelled -- per story 32.
Good idea about the enum -- Story 30 looks like a good place in the workflow to implement it